irmasTrainUtils.py

```diff
@@ -78,10 +83,15 @@ def parse_irmas_trainset (source, dest):
             augmentations = augment_song(rate, data)
             
             # Write the song to DEST
+            destDir = dest + "/Train"
+            if i > len(files) * percentage:
```

Comment:
Maybe use randomization instead? Otherwise I think this looks good

Reply:
I think randomization is unnecessary since the songs themselves are pretty much random in their classes. The code as of now splits 90/10 based on each class, meaning that the train and validation sets should have an effectively random distribution of music with the same percentage belonging to each class.